src/pages/GuestInformationForm.jsx

```
@@ -0,0 +1,591 @@
+      } else {
+        console.error("Submission error:", error);
+        toast.error("Something went wrong, plese try again later");
+      }
+    } finally {
+      hideLoading();
+    }
+  };
+
+  const verifyGST = async (event) => {
+    event.preventDefault();
+    const url = `https://api.gst.gov.in/v1/verify?gstin=${formData.gstNumber}`;
```

ESLint

[error] 'url' is assigned a value but never used.

(no-unused-vars)